src/apps/dfx/ethereum/dfx.balance-fetcher.ts

```diff
@@ -0,0 +1,38 @@
+// Test address: http://localhost:5001/apps/dfx/balances?addresses[]=0x1867608e55a862e96e468b51dc6983bca8688f3d&network=ethereum
+//
+import { Inject } from '@nestjs/common';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { presentBalanceFetcherResponse } from '~app-toolkit/helpers/presentation/balance-fetcher-response.present';
+import { BalanceFetcher } from '~balance/balance-fetcher.interface';
+import { Network } from '~types/network.interface';
+
+import { DFX_DEFINITION } from '../dfx.definition';
+
+const network = Network.ETHEREUM_MAINNET;
+
+@Register.BalanceFetcher(DFX_DEFINITION.id, network)
+export class EthereumDfxBalanceFetcher implements BalanceFetcher {
+  constructor(@Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit) {}
+
+  async getCurveTokenBalances(address: string) {
+    return this.appToolkit.helpers.tokenBalanceHelper.getTokenBalances({
+      address,
+      appId: DFX_DEFINITION.id,
+      groupId: DFX_DEFINITION.groups.curve.id,
+      network,
+    });
+  }
+
+  async getBalances(address: string) {
+    const [curveTokenBalances] = await Promise.all([this.getCurveTokenBalances(address)]);
```

Comment:
You probably also want to add `getStakingBalances` here, and use the `ContractPositionBalanceHelper` (on `this.appToolkit.helpers.contractPositionBalanceHelper`) to build the staked balances.

Reply:
Fix https://github.com/Zapper-fi/studio/pull/347/commits/34713400bc9a836d3d19929fb6438d9ee8c86d32